site-scrapers/Hannaford.js

```diff
@@ -63,10 +65,77 @@ async function ScrapeWebsiteData(browser) {
                 const filename =
                     "hannaford-zip-" + loc.zip + "-date-" + today + ".png";
                 await page.screenshot({ path: filename });
+                await saveBodyHtml(page);
```

Comment:
oh also I think we should only do this if we're not in DEVELOPMENT mode. So either skip saving the page & screenshot entirely if in dev mode, or only save it locally

Also, wondering if we should also save the screenshot to S3 in production?

Reply:
Sounds good.